src/reporting/claim-reporting-fees.js

```diff
@@ -221,33 +253,37 @@ function redeemContractFees(p, payload, successfulTransactions, failedTransactio
 }
 
 /**
- * TODO: Add updated JSDoc info for input/returned values.
- *
  * Claims all reporting fees for a user as follows:
  *
- * If the forked market is finalized:
- *   Call `Market.migrateThroughOneFork` for all non-forked, non-finalized non-migrated markets in the same universe as the forked market.
- * Else:
- *   Call `Market.disavowCrowdsourcers` for all non-forked, non-finalized, non-disavowed markets in the same universe as the forked market.
+ * If a forked market exists in the current universe:
+ *   For all non-finalized markets in current universe where the user has unclaimed fees in the reporting participants:
+ *     Call `Market.disavowCrowdsourcers`
  *
- * Once the above transactions are finished:
- *   Call `FeeWindow.redeem` on all specified FeeWindows
- *   For each DisputeCrowdsourcer the user unredeemed staked in:
- *     If its market is forked and `DisputeCrowdsourcer.fork` has not been called:
- *       Call `DisputeCrowdsourcer.forkAndRedeem`
- *     Else:
- *       Call `DisputeCrowdsourcer.redeem`
- *   For each InitialReporter the user has unredeemed stake in:
- *     If its market is forked and `InitialReporter.fork` has not been called:
- *       Call `InitialReporter.forkAndRedeem`
+ * Once the above has been completed:
+ *   Call `FeeWindow.redeem` on all fee windows in the current universe where the user has unclaimed participation tokens
+ *   For each market participant of the forked market in which the user has unclaimed fees:
+ *     If `DisputeCrowdsourcer.fork`/`InitialReporter.fork` has not been called:
+ *       Call `DisputeCrowdsourcer.forkAndRedeem`/`InitialReporter.forkAndRedeem`
  *     Else:
- *       Call `InitialReporter.redeem`
+ *       Call `DisputeCrowdsourcer.redeem`/`InitialReporter.redeem`
+ *   For all other market participants:
+ *     Call `DisputeCrowdsourcer.redeem`/`InitialReporter.redeem`
```

Comment:
Is this in reference to the forking market or non-forking markets?

Reply:
It's in reference to all non-forked-market reporting participants. (Though this comment section is going to change in my next PR because I'm creating a separate function just for claiming forked-market reporting participants, and this one will just be for claiming non-forked-market reporting participants.)

Reply:
InitialReporters are not redeemed on non-forked markets

Reply:
They're not? How else do they get redeemed?

Reply:
When the horror show is over. They are kept attached to the market, and come along in the migrateThroughOneFork to the winning universe.

Reply:
Wait, but shouldn't the Claim button be claiming them if, for example, there is no forked market in the current universe?

Reply:
https://github.com/AugurProject/augur-core/blob/master/source/contracts/reporting/Market.sol#L311
When the non-forking market disavows (OR migrateThroughOneFork) is disavows ALL things, but then reattaches the initial

Reply:
Yes, sorry, this line is in the context of no fork at all? then it is true , yea, sorry i wasn't sure the context of the line. We need new terms:
forking-market = forking market
non-forking-market = non-forking market in a universe with a forking market
market = regulr

Reply:
When i hear "non-forking-market" i've been thinking that it implies a fork is going on

Reply:
Well, this line is in the context of Market.disavowCrowdsourcers having already been called on non-forked markets, depending on if there's a fork in the current universe (see a few lines above).  Then, regardless of whether there's a fork in the current universe, redeem gets called on the non-forked reporting participants.

Reply:
In any event, I'll go ahead and merge this.  This comment is updated in my next PR anyway. :)

Reply:
If market.disavowCrowdsourcers was called on the non-forked market, you do not claim the initial reporter, is all i'm saying

Reply:
Ah, right. Sorry, was getting confused on crowdsourcers vs initial reporter.